pkg/ruby/gemspec/parse.go

```diff
@@ -39,7 +38,7 @@ var (
 	// Capture the value of "licenses"
 	// e.g. s.license = ["MIT".freeze, "BSDL".freeze]
 	//      => "MIT".freeze, "BSDL".freeze
-	licensesRegexp = regexp.MustCompile(`\.licenses\s*=\s*\[(?P<licenses>.+)\]`)
```

Comment:
Do we need to remove this `\`?

Reply:
This `\` is redundant, because regex doesn't have character set opener (`[`).
Perhaps this can be confuse.
Do you want me to return `/`?

Reply:
> regex doesn't have character set opener ([).

`[]` has a special meaning in regexp, so I thought we needed to escape them like `\[\]`.
https://github.com/google/re2/wiki/Syntax

Actually, an opener is still escaped `\[`. Is that escape also unneeded?
`\.licenses\s*=\s*[(?P<licenses>.+)]`

Reply:
My IDE(`goland`) recommended me to remove redundant symbols.
I checked it with [regex101](https://regex101.com/r/rCuT62/1). It really works.
But I didn't find any explanation why it works.

Looks like there are 2 cases:
- regex contain `[`. Then `]` will complete character set(e.g. [[cba]](https://regex101.com/r/CjZXNs/2))
- regex doesn't contain `[` (or this character is escaped). Then `]` will be common character(e.g. [cba\]](https://regex101.com/r/603dQJ/1) or [\\\[cba\]](https://regex101.com/r/603dQJ/2))

Reply:
Interesting. Yes, we should escape `[` and `]` both for clarity.